tests/test_init.py

```diff
@@ -53,3 +60,102 @@ def test_parse_bad_patchwork_url(self):
             "Can't parse patchwork url: '{}'".format(patchwork_url)
             in context.exception
         )
+
+
+class KBuilderTest(unittest.TestCase):  # pylint: disable=R0902
+    """Test cases for skt.kbuilder class"""
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
+        self.tmpconfig = tempfile.NamedTemporaryFile()
+        self.kbuilder = skt.kbuilder(self.tmpdir, self.tmpconfig.name)
+        self.m_popen = Mock()
+        self.m_popen.returncode = 0
+        self.ctx_popen = mock.patch('subprocess.Popen',
+                                    Mock(return_value=self.m_popen))
+        self.ctx_check_call = mock.patch('subprocess.check_call', Mock())
+        self.m_io_open = Mock()
+        self.m_io_open.__enter__ = lambda *args: self.m_io_open
+        self.m_io_open.__exit__ = lambda *args: None
+        self.ctx_io_open = mock.patch('io.open',
+                                      Mock(return_value=self.m_io_open))
```

Comment:
I'm concerned about the approach of having instance variables shared and updated by tests. This is similar to having global variables. I think this creates an opportunity for a test to use one of those without updating it, and then breaking when tests are reordered or omitted. Do you think you can instead, e.g., write functions creating those values as needed and then use them in the tests, or just directly creating them every time?